src/components/styles/style.css

```diff
@@ -1,12 +1,23 @@
-ul {
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@500&display=swap');
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+li {
   list-style-type: none;
+  margin-top: 15px;
+}
+
+tr:nth-child(even) {
```

Comment:
- Great Job for updating your design based given design👍 I think it would be better if you update the cards title position to the right bottom, add an arrow on the right top and remove the border of the card to stick the given design, and have a better UI👌

| Your design                                                             | Given design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/236655753-c790ab7b-b3c2-4b3b-9a07-96f57c4e9a8e.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/236655755-e83032ef-09a8-4431-9da3-a9ebd66a339c.png" width="200">|


😎️

🤝️